WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -10845,7 +10845,7 @@
 			path = Classes;
 			sourceTree = "<group>";
 		};
-		29B97314FDCFA39411CA2CEA = {
+		29B97314FDCFA39411CA2CEA /* CustomTemplate */ = {
```

Comment:
All these file format changes every time CocoaPods touches the project and then Xcode touches is again... 🙄 

I wonder if there's a way to address this. Or maybe it's just a minor annoyance we (_I_) need to learn to put up with.